documentation/modules/configuring/con-config-kafka-connect.adoc

```diff
@@ -37,27 +37,34 @@ For more information, see xref:con-high-volume-config-properties-{context}[Handl
 .Example `KafkaConnect` custom resource configuration
 [source,yaml,subs=attributes+,options="nowrap"]
 ----
+# Basic configuration (essential)
 apiVersion: {KafkaConnectApiVersion}
 kind: KafkaConnect # <1>
 metadata:
   name: my-connect-cluster
   annotations:
     strimzi.io/use-connector-resources: "true" # <2>
+# Deployment specifications
 spec:
+  # Replicas (essential)
   replicas: 3 # <3>
+  # Authentication (recommended)
```

Comment:
Same as in the Bridge. I think the `authentication` and `tls` section should be the same - either both recommended or both optional.